exchanges/bitstamp/bitstamp_type_convert.go

```diff
@@ -0,0 +1,30 @@
+package bitstamp
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// UnmarshalJSON deserializes JSON, and timestamp information.
+func (p *TradingPair) UnmarshalJSON(data []byte) error {
+	type Alias TradingPair
+	t := &struct {
+		*Alias
+		MinimumOrder string `json:"minimum_order"`
+	}{
+		Alias: (*Alias)(p),
+	}
+
+	err := json.Unmarshal(data, t)
+	if err != nil {
+		return err
+	}
+	before, _, found := strings.Cut(t.MinimumOrder, " ")
+	if !found {
+		return fmt.Errorf("unhandled minimum order string: %s", t.MinimumOrder)
+	}
+	p.MinimumOrder, err = strconv.ParseFloat(before, 64)
```

Comment:
The currency suffix isn't documented, ergo could be subject to change.
If we use Cut like this then I think we should handle t.MinimumOrder as a fallback when not found.
```suggestion
	minOrderStr := t.MinimumOrder
	if prefix, _, found := strings.Cut(t.MinimumOrder, " "); found {
		minOrderStr = prefix
	}
	p.MinimumOrder, err = strconv.ParseFloat(minOrderStr, 64)
```

Reply:
@shazbert Feedback on this proposal ☝️ please

Reply:
@gbjk Looks good, nice suggestion.